deployment-script/ConfiguredScript.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../lib/forge-std/src/Script.sol";
+import "../lib/forge-std/src/console2.sol";
+
+abstract contract ConfiguredScript is Script {
+    using stdJson for string;
+
+    bool internal immutable SAVE_VERIFY = true;
+
+    string internal configPath;
+
+    address internal morphoAddressEthereum;
+
+    function _init(string memory network) internal returns (bytes memory) {
+        vm.createSelectFork(vm.rpcUrl(network));
```

Comment:
We are missing the alias for base: we need to add the corresponding definition in the `[profile.default.rpc_endpoints]` section of `foundry.toml`

Reply:
Right